lib/resolve.js

```diff
@@ -0,0 +1,121 @@
+import fs from "fs";
+import path from "path";
+import _ from "lodash";
+
+
+const WINDOWS = process.platform === "win32";
```

Comment:
As far as I can remember, the `path` module should suffice everywhere else.  This is special because of how it is walking up the directory hierarchy.  But it'd be worth taking a pass over all the code base to verify...